backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -537,7 +537,7 @@ export class UserResolver {
     await queryRunner.startTransaction('READ UNCOMMITTED')
 
     try {
-      if (coinanimation) {
+      if (coinanimation !== undefined) {
```

Comment:
Why?

Reply:
```
null !== undefined

-> true
```

Reply:
```
if (undefined)
 -> false
```

Reply:
```
if (null)
 -> false
```

Reply:
Yeah cause of: 
```
if (false)
  -> store false
```

Reply:
Now it does not store false, so each time you log back in you get the coin animation, the db update is not triggered on coinanimation === false!

Reply:
```suggestion
      if (coinanimation !== null && coinanimation !== undefined) {
```